src/routes/Routes.jsx

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import { Route, Routes } from "react-router-dom";
-import { Mockbee, MockMan,Home,ProductDisplay } from "pages";
+import { Mockbee, MockMan, Home, ProductDisplay, SignIn, SignUp, Cart, Wishlist } from "pages";
+import { ProtectedRoutes } from "./ProtectedRoutes";
+
 const RoutesList = () => {
     return (
         <Routes>
-            <Route path="/" element={<Home/>}/>
-            <Route path="/products" element={<ProductDisplay/>}/>
+            <Route path="/" element={<Home />} />
+            <Route path="/products" element={<ProductDisplay />} />
             <Route path="/mockbee" element={<Mockbee />} />
             <Route path="/mockman" element={<MockMan />} />
+            <Route path="/signin" element={<SignIn />}></Route>
+            <Route path="/signup" element={<SignUp />}></Route>
+            <Route path="/cart" element={<ProtectedRoutes specificPart={<Cart />}></ProtectedRoutes>}></Route>
```

Comment:
I didn't know how to implement these private routes but looking at your code really helped